### ISSUE TITLE
Add documentation

### ISSUE THREAD

**Issue by sarthak-sopho** [elixir-cloud-aai/foca#6]
- Write a README.md that describes what the package is about.
- Put code coverage service badge in docs

**Comment by sarthak-sopho** [comment]
Put code coverag service badge in docs

**Comment by uniqueg** [comment]
I've added the project to [Coveralls](https://coveralls.io). Please add the following badge from [shields.io](https://shields.io/category/coverage) rather than the one from Coveralls itself. This is the markdown code (check the `README.md` in the [TEStribute](https://github.com/elixir-cloud-aai/TEStribute) project for comparison):

```md
[![Coverage](https://img.shields.io/coveralls/github/elixir-cloud-aai/archetype-flask-connexion)](https://coveralls.io/github/elixir-cloud-aai/archetype-flask-connexion)
```

It creates a clickable badge that points to the detailed coverage report on Coveralls (obviously, it doesn't yet report any coverage):
[![Coverage](https://img.shields.io/coveralls/github/elixir-cloud-aai/archetype-flask-connexion)](https://coveralls.io/github/elixir-cloud-aai/archetype-flask-connexion)

**Comment by vaniisgh** [comment]
Hey! In case you haven't begun writing the readme I would like to take this on. :)

**Comment by sarthak-sopho** [comment]
Hey! Yeah, please go ahead.

**Comment by uniqueg** [comment]
yes, @vaniisgh, would be great to have you on board for this :)

**Comment by uniqueg** [comment]
Closed by #22 

Sorry @vaniisgh, just added this quickly while wrapping up the packaging for PyPI.

From now on, please always add any documentation, along with all unit and other necessary tests, for each PR
